frappe/model/document.py

```diff
@@ -1148,12 +1148,12 @@ def composer(self, *args, **kwargs):
 
 		return composer
 
-	def is_whitelisted(self, method):
-		fn = getattr(self, method, None)
+	def is_whitelisted(self, method_name):
+		method = getattr(self, method_name, None)
 		if not fn:
```

Comment:
I guess it should be `if not  method`

Reply:
Sorry, will fix.

Reply:
I have also added a small summary of changes made.